app/models/katello/concerns/organization_extensions.rb

```diff
@@ -182,6 +182,17 @@ def cancel_repo_discovery
         def regenerate_ueber_cert
           ::Katello::Resources::Candlepin::Owner.generate_ueber_cert(self.label)
         end
+
+        def expiring_subscriptions
+          subscriptions.select(&:expiring_soon?)
+        end
+
+        def notification_recipients_ids
+          users = User.unscoped.all.find_all do |user|
+            user.can?(:import_manifest) && user.can?(:view_organizations, self)
```

Comment:
@jyejare The problem I had was that previously I was calling user.can?(:import_manifest, self) here. This would not work, because import_manifest is a permission for Subscription, not for Organization. Hence, only admins would return 'true' for that.

tl;dr - It should be fine now for org admins